tests/integration/utl_wal_version_test.go

```diff
@@ -85,7 +68,7 @@ func TestEtcdVersionFromWAL(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	assert.Equal(t, &semver.Version{Major: 3, Minor: 6}, walVersion.MinimalEtcdVersion())
+	assert.Equal(t, &semver.Version{Major: 3, Minor: 5}, walVersion.MinimalEtcdVersion())
```

Comment:
I think it's ok to leave this test and have it return v3.4 as minimal version.

Reply:
I will update the proto (add a new message/field) for testing only in a followup PR, and then I will rewrite this test by then.

Reply:
Updated.